Description: Use the "readthedocs" mkdocs theme instead of the "material" one
 There are some issues with invoking `dh_mkdocs` for a non-default theme.
Forwarded: not-needed
Author: Peter Pentchev <pp@storpool.com>
Last-Update: 2023-06-04

--- a/mkdocs.yml
+++ b/mkdocs.yml
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: BSD-2-Clause
 
 theme:
-  name: material
+  name: readthedocs
   features:
     - navigation.instant
     - navigation.tracking
